libexec/fixup.ts

```diff
@@ -20,31 +20,31 @@ const pkg_prefix = new Path(unknown[0])
 
 switch (host().platform) {
 case 'darwin': {
-  const { success } = await Deno.run({
-    cmd: [
+  const { output } = new Deno.Command(Deno.execPath(), {
```

Comment:
should be `fix-machos.rb`.

It's interesting this works, I guess that deno re-execs shebangs? To be clear this runs the ruby script through `deno`!

Reply:
Sorry not following, what should be fix-machos.rb?
fix-machos.rb is run on line 25 no?

Reply:
You are executing `deno`, this is what `Deno.execPath()` returns. Replace it with `fix-machos.rb`

Reply:
Ah I see.
Couldn't tell in [deno.run docs](https://deno.land/api@v1.37.2?s=Deno.run) if line 25 and 26 are args or not but saw my mistake with the actual command. Built a package with this and it completely successfully